### PR TITLE
Make csproj msbuild projects buildable.

### PR DESCRIPTION
**Pull request by powercode**
Made the projects in PowerShell.sln build and run.

This makes it easier to develop and debug changes, even if it doesn't work for Linux/macOs.
Copying a powershell.version file and setting $env:devpath to src/vs-proj/bin/release before launching devenv make debugging work as expected from visual studio.

### CODE REVIEW

File: src/cimSupport/cmdletization/xml/cmdlets-over-objects.xsd [hunk 1]
```diff
@@ -0,0 +1,1050 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (vors) [inline src/cimSupport/cmdletization/xml/cmdlets-over-objects.xsd]: Did you copy it from win 10 box?
We should check that it's the latest version.

Comment (powercode) [inline src/cimSupport/cmdletization/xml/cmdlets-over-objects.xsd]: Yes, from a Win10 Anniversary

Comment (powercode) [inline src/cimSupport/cmdletization/xml/cmdlets-over-objects.xsd]: I changed the default build of SMA to Debug. Makes for a much better debugging experience.
This change would be really nice to have merged. It makes working on the project much nicer.

**Comment by lzybkr** [comment]
We're moving to official msbuild support in .Net soon, so we won't be taking this PR.

**Pull request closed without merging**